packages/components/namespace-store/create-ns.tsx

```diff
@@ -1,14 +1,63 @@
-import * as React from 'react';
-import { RouteComponentProps } from 'react-router';
-
-const CreateNamespaceStore: React.FC<CreateNamespaceStoreProps> = ({ match }) => {
-    return (
-        <div className="center-component">
-            <>{"Data Federation ~ Sample Form"}</>
-        </div>
-    );
+import * as React from "react";
+import { useFlag } from "@openshift-console/dynamic-plugin-sdk";
+import { useTranslation } from "react-i18next";
+import { RouteComponentProps } from "react-router";
+import "../noobaa-provider-endpoints/noobaa-provider-endpoints.scss";
+import { Title } from "@patternfly/react-core";
+import { CEPH_STORAGE_NAMESPACE, ODF_MODEL_FLAG } from "../../constants";
+import { NooBaaNamespaceStoreModel } from "../../models";
+import { getName } from "../../shared/selectors/k8s";
+import { referenceForModel } from "../../shared/utils/common";
+import NamespaceStoreForm from "./namespace-store-form";
+
+const CreateNamespaceStore: React.FC<CreateNamespaceStoreProps> = ({
+  history,
+  match
+}) => {
+  const { t } = useTranslation('plugin__dfr-console');
+  const { ns = CEPH_STORAGE_NAMESPACE, appName } = match.params;
```

Comment:
```suggestion
  const { ns = DATA_FEDERATION_NAMESPACE, appName } = match.params;
```

Reply:
similarly everywhere

Reply:
Done already, just missed pushing it to the remote repository.